openedge-plugin/src/main/java/org/sonar/plugins/openedge/foundation/OpenEdgeSettings.java

```diff
@@ -195,7 +194,7 @@ private final void initializeXrefBytes(Settings settings) {
     }
   }
 
-  private final void parseBuildDirectory() {
+  public final void parseBuildDirectory() {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Refactor this method to reduce its Cognitive Complexity from 17 to the 15 allowed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar.riverside-software.fr/coding_rules#rule_key=squid%3AS3776)